.github/stale.yml

```diff
@@ -9,6 +9,12 @@ exemptLabels:
   - reliability
   - release requirement
 
+# Number of days of inactivity before stale
+daysUntilStale: 60
+
+# Number of days of inactivity before close
+daysUntilClose: 30
```

Comment:
@kuja53 can we reduce this to 7 please?

Reply:
This need to be reduced total 90 days seems high